documents/functionnal_specifications/functional.md

```diff
@@ -0,0 +1,445 @@
+  - **Alert Notifications**:
+    - Warns players of critical issues such as **natural disasters**, **political unrest**, or **scientific breakthroughs**.
+
+**Integration of GDP and Research**
+
+- GDP acts as the **financial backbone**:
+  - Players must maintain GDP to fund **research** while minimizing harm to the environment.
+  - Excessive industrial expansion may increase GDP temporarily but lead to higher **emissions** and **ecological degradation**.
+- The **scientific research system** challenges players to:
+  - Prioritize research areas based on current global challenges.
+  - Balance short-term GDP sacrifices with long-term sustainability goals. 
+
+
+
+### Pin & Icon System
+
+#### 1. Design
+---
+
+We have 4 categories :
+
+We have defined five main categories in agriculture, each representing a key area that affects the ecological status of a zone. These categories can be upgraded to reduce environmental impact and improve sustainability.
+
+| Agriculture | Description | Icon |
+|-------------|-------------|------|
+| **Livestock** | Represents the impact of livestock farming on the environment, including greenhouse gas emissions, water usage, and land degradation. Upgrades in this category may focus on sustainable livestock practices, waste reduction, and improved animal welfare. | ![godot](pictures/agriculture/icons8-panneau-bovins-60.png) |
+**Crops**            | Covers the production and management of various crops. The focus is on soil health, pesticide use, water conservation, and crop rotation. Upgrades here can lead to reduced chemical runoff and improved resilience to climate change. | ![godot](pictures/agriculture/icons8-champ-50.png) |
+
+<br>
+
+| Transport    | Description                                                                                                                                    | Icon                                             |
+|--------------|------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------|
+| **Car**      | Represents the environmental impact of personal and small vehicle transportation, including emissions and fuel consumption. Upgrades may focus on reducing carbon output, promoting electric vehicles, and increasing fuel efficiency. | ![car](pictures/transport/icons8-berline-60.png) |
+| **Bus/Truck** | Encompasses larger land vehicles used for public and freight transportation. This category emphasizes emissions reduction, fuel optimization, and transitioning to sustainable energy sources. | ![bus_truck](pictures/transport/icons8-navette-50.png) |
+| **Subway**   | Represents underground or metropolitan rail systems, which generally offer lower emissions per passenger. Improvements in this category could enhance energy efficiency and optimize route planning for reduced impact. | ![subway](pictures/transport/icons8-métro-48.png) |
```

Comment:
345 : replace `systems` with `networks`;  `for reduced impact` with `to reduce impacts`